demo.html

```diff
@@ -11,11 +11,11 @@
   <link rel="stylesheet" type="text/css" href="assets/css/locuszoom.css"/>
 
   <script type="text/javascript">
-    LocusZoom.DefaultDataSources = {
-        position: new LocusZoom.Data.AssociationSource("http://portaldev.sph.umich.edu/api_internal_dev/v1/single/"),
-        ld: new LocusZoom.Data.LDSource("http://portaldev.sph.umich.edu/api_internal_dev/v1/pair/LD/"),
-        gene: new LocusZoom.Data.GeneSource("http://portaldev.sph.umich.edu/api_internal_dev/v1/annotation/genes/")
-    };
+    var apiBase = "http://portaldev.sph.umich.edu/api_internal_dev/v1/";
+    var ds = new LocusZoom.DataSources();
+	ds.addSource("base", ["AssociationLZ", {url:apiBase + "single/", params: {analysis: 3}}]);
```

Comment:
I think it will be more clear how everything ties together when we extract out the layout object since that will directory connect to the datasource names. Right now I didn't change too much so as to not break too many things. The other big thing we need to rename is "Instance." That sticks out like a sore thumb to me when we talk about an "instance of Instance". Right now i'm leaning more toward "Plot". It's a bit more generic than i like but i think it's better than abusing "instance." Naming is hard.